cap/modules/experiments/static/js/experiments/controllers.js

```diff
@@ -0,0 +1,24 @@
+define([], function() {
+  var controllers = angular.module('cap.experiments.controllers', [])
+    .controller('experimentsController', [
+      '$scope',
+      'capLocalClient',
+       function($scope, capLocalClient) {
+        $scope.hello = 'CAP experiments';
+        $scope.notification = 'Welcome to CAP experiments';
```

Comment:
Can you please change "Welcome to CAP" to "Welcome to CERN Analysis Preservation portal" here and everywhere?  The abbreviation CAP is internal, we don't want to advertise it...